doc/rst/usingchapel/building.rst

```diff
@@ -135,6 +135,10 @@ Specifically:
   from the Chapel source tree into ``/dir/for/install``, preserving
   Chapel's traditional directory structure.
 
+* Please note that for any installed version of Chapel, the meaning of
+  ``$CHPL_HOME`` as printed by compiler messages refers to
+  ``/dir/for/install/share/chapel/x.yz`` where ``x.yz`` is the Chapel
+  version being installed.
```

Comment:
This new documentation note is only accurate for installs done after `./configure --prefix=...` and it is not correct for installs done with `./configure --chpl-home=...`. As such, I recommend you put it in that bullet (which is starting at line 121) instead of in its own bullet.

Reply:
Shoot, I was obviously reading too fast, thanks!

Reply:
@mppf: I've moved the text, wordsmithed epsilon of it, and added a similar sentence to the `--chpl-home` case if you'd be up for giving it a quick look.